geomstats/learning/geometric_median.py

```diff
@@ -0,0 +1,110 @@
+"""Geometric Median Estimation."""
+
+import geomstats.backend as gs
+
+
+class WeiszfeldAlgorithm:
```

Comment:
Please call this GeometricMedian, as we need name match between:
- the name of the file, here geometric_median.py
- the name used in the tests, should be TestGeometricMedian.

Just explain in the docstring that the geometric median is computed with the weiszfeld algorithm.